src/widgets/BaseWindow.cpp

```diff
@@ -23,6 +23,7 @@
 
 #ifdef USEWINSDK
 #    include <dwmapi.h>
+#    include <shellapi.h>
```

Comment:
Unrelated change, but it doesn't compile without it. Something related to `expected` probably, but I can't imagine how.